packages/back-end/src/routers/safe-rollout/safe-rollout.helper.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  FeatureInterface,
+  FeatureRule,
+  SafeRolloutRule,
+} from "back-end/src/validators/features";
+
+export function getSafeRolloutRuleFromFeature(
+  feature: FeatureInterface,
+  ruleId: string
+): SafeRolloutRule | null {
+  Object.keys(feature.environmentSettings).forEach((env: string) =>
+    feature.environmentSettings[env].rules.forEach((rule: FeatureRule) => {
+      if (rule.id === ruleId) {
```

Comment:
If we're strictly typing this function to return a `SafeRolloutRule`, we should verify the type here instead of just relying on the id matching.